lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -125,10 +125,26 @@ def add_amount(post, amount)
         }
       end
 
+      def add_ach(post, payment)
+        post[:paymentInformation][:bank] = {
+          account: {
+            type: ach_account_type(payment.account_type),
+            number: payment.account_number
+          },
+          routingNumber: payment.routing_number
+        }
+      end
+
+      def ach_account_type(account_type)
+        account_type == 'checking' ? 'C' : 'S'
+      end
```

Comment:
Very minor point but let's move these methods below the `add_payment` method that calls them